docopt_api.h

```diff
@@ -0,0 +1,37 @@
+//
+//  docopt.h
+//  docopt
+//
+//  Created by Dmitriy Vetutnev on 2019-09-05.
+//  Copyright (c) 2019 Dmitriy Vetutnev. All rights reserved.
```

Comment:
Can we remove the attribution and copyright here? I don't think we necessarily need _any_ copyright attribution outside of the top one. (In other words, I'm not asking you to change it match the docopt.h header, just leave only the lines 1-4 and delete 5 and 6).

Reply:
Ok, this lines removed